libs/designer/src/lib/ui/panel/templatePanel/createWorkflowPanel/tabs/nameStateTab.tsx

```diff
@@ -192,11 +191,7 @@ export const nameStateTab = (
     id: 'BX0M13',
     description: 'The tab label for the monitoring name and state tab on the create workflow panel',
   }),
-  description: intl.formatMessage({
-    defaultMessage: 'Provide a unique, descriptive name and review the state type to ensure your workflow is properly configured.',
-    id: 'jgy58N',
-    description: 'An accessibility label that describes the objective of name and state tab',
-  }),
+  description: undefined,
```

Comment:
Do we need to specify a undefined value, shouldnt not specifying it means the same?

Reply:
I initially wanted to specify there so that we know this tab doesn't have description, but just removed it!g